core/src/main/java/software/amazon/awssdk/retry/RetryPolicyContext.java

```diff
@@ -59,8 +58,7 @@ public Object originalRequest() {
     }
 
     /**
-     * @return The marshalled request. See {@link Request#addHandlerContext(HandlerContextKey, Object)} for a mechanism to store
-     *     request level state across invocations of the retry policy.
+     * @return The marshalled request.
```

Comment:
Should we include ExecutionContext in here so they have somewhere to store mutable state?